#!/usr/bin/env python
PACKAGE = "snake_robot"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("use_gui",   bool_t,   0, "sets_gui_control",  False)

gen.add("j_1",double_t,0, "joint1",0,-3.1,3.1)
gen.add("j_2",double_t,0, "joint2",0,-3.1,3.1)
gen.add("j_3",double_t,0, "joint3",0,-3.1,3.1)
gen.add("j_4",double_t,0, "joint4",0,-3.1,3.1)
gen.add("j_5",double_t,0, "joint5",0,-3.1,3.1)
gen.add("j_6",double_t,0, "joint6",0,-3.1,3.1)
gen.add("j_7",double_t,0, "joint7",0,-3.1,3.1)
gen.add("j_8",double_t,0, "joint8",0,-3.1,3.1)
gen.add("j_9",double_t,0, "joint9",0,-3.1,3.1)
gen.add("j_10",double_t,0, "joint10",0,-3.1,3.1)
gen.add("j_11",double_t,0, "joint11",0,-3.1,3.1)
gen.add("j_12",double_t,0, "joint12",0,-3.1,3.1)
gen.add("j_13",double_t,0, "joint13",0,-3.1,3.1)
gen.add("j_14",double_t,0, "joint14",0,-3.1,3.1)
gen.add("j_15",double_t,0, "joint15",0,-3.1,3.1)
exit(gen.generate(PACKAGE, "amcl_correction", "Params"))
